ui/src/app/utils/resource-list.tsx

```diff
@@ -83,9 +103,11 @@ export const ResourcesList = fp.flow(withCdrVersions())((props: Props) => {
                 {getDisplayName(r)}
               </ResourceNavigation>
             ),
+            name: getDisplayName(r),
```

Comment:
This is too similar to the other field, so I'd prefer "resourceNameAsString" or something like that